lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -91,16 +71,14 @@ namespace :webpack do
     end
   end
 
-  task :paths => 'ui:load_inflectors' do
-    paths_config = ManageIQ::UI::Classic::Engine.root.join('config/webpack/paths.json')
+  task :paths do
+    json = JSON.pretty_generate(
+      :info    => "This file is autogenerated by rake webpack:paths, do not edit",
+      :output  => Rails.root.to_s,
+      :engines => asset_engines.map { |p| [p.webpack_name, p.path] }.to_h
```

Comment:
Surprised you didn't make this into a helper method on the `ManageIQ` side.

Reply:
Should https://github.com/ManageIQ/manageiq-ui-classic/pull/3776 get merged first, this will need to change so that the result looks like...

```

```

instead of 

```
...
    "manageiq-graphql": {
      "root": "/home/himdel/.rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/bundler/gems/manageiq-graphql-5f68621f2791",
      "node_modules": "/home/himdel/.rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/bundler/gems/manageiq-graphql-5f68621f2791/node_modules"
    },
...
```

instead of 

```
...
   "manageiq-graphql": "/home/himdel/.rbenv/versions/2.4.3/lib/ruby/gems/2.4.0/bundler/gems/manageiq-graphql-5f68621f2791",
...
```

Reply:
That should be an easy change, and could be done either here or in the AssetPath class in core.

Reply:
> Surprised you didn't make this into a helper method on the ManageIQ side.

I felt like the ManageIQ side should just be responsible for the attributes (like a glorified hash), but not how to *use* the attributes.

Reply:
@himdel Do you have a preference on which to merge first (provided the other one goes green)?  This one can be merged separate from ManageIQ/manageiq#17643 (i.e. it doesn't need a simultaneous merge)

Reply:
Either way works for me, but I will be on PTO since Friday, so.. if we merge this today, I can still fix mine, if not, it would have to wait.

Reply:
> I felt like the ManageIQ side should just be responsible for the attributes (like a glorified hash), but not how to use the attributes.

Cool.  That works for me.